sync_radar_data_globus.py

```diff
@@ -171,8 +172,37 @@ def synchronize(self):
                 pass
             else:
                 pass
-            listing = self.transfer_client.operation_ls(self.mirror_uuid, path=listing_path,
-                                                        filter=listing_pattern)
+            print("Listing path: {path} on endpoint: {endpoint} with pattern: {pattern}".format(
+                path=listing_path, endpoint=self.mirror_uuid, pattern=listing_pattern))
+            listing_succeeded = False
+            operation_ls_retries = 10 
+            attempts = 0
+            listing = []
+            listing_times = []
+
+            while attempts < operation_ls_retries:
+                try:
+                    listing_times.append(time.time())
+                    print("Attempt: {}".format(attempts)),
+                    listing = self.transfer_client.operation_ls(self.mirror_uuid, path=listing_path,
+                                                                filter=listing_pattern)
+                    listing_succeeded = True
+                    break
+                except globus_sdk.GlobusAPIError:
+                    print("... failed")
```

Comment:
Would it be helpful to `print('...successful')` if it succeeds as well?

Reply:
I think I'll take out the ...failed statement and only print a dot every time it retries, then print 'failed' if it runs out of retries,

Reply:
Sounds good to me, what about success?

Reply:
I think that success is implied if it moves on, so don't think it's necessary